bowling/test/input-ut.cpp

```diff
@@ -0,0 +1,74 @@
+#include <gtest/gtest.h>
+
+#include "input.hpp"
+
+constexpr char NON_EXISTING_PATH[] = "nonexistingPath";
+constexpr char INPUT_DIRECTORY[] = "../test/inputTestDirectory";
+constexpr size_t INPUT_DIRECTORY_FILES_COUNT = 3;
+
+constexpr char LANE1_NAME[] = "lane1";
+constexpr char LANE2_NAME[] = "lane2";
+constexpr char LANE3_NAME[] = "lane3";
+
+constexpr size_t LANE1_PLAYERS_NUM = 3;
+constexpr size_t LANE2_PLAYERS_NUM = 0;
+constexpr size_t LANE3_PLAYERS_NUM = 2;
+
+constexpr char LANE1_PLAYER1[] = "Name1:X|4-|3";
+constexpr char LANE1_PLAYER2[] = "Name2:34|X|0-";
+constexpr char LANE1_PLAYER3[] = ":X|22|33";
+
+TEST(InputTest, ShouldNotValidateWrongPath) {
+    //GIVEN
+    std::string path = NON_EXISTING_PATH;
+
+    //WHEN
+    Input input(path);
```

Comment:
then you can youse your class here:
```suggestion
TEST_F(YourClass, ShouldNotValidateWrongPath) {
    //WHEN
    Input input(NON_EXISTING_PATH);
```

Reply:
ditto